src/app/(home)/mainPage.module.scss

```diff
@@ -0,0 +1,25 @@
+.mainPageWrap {
+	display: flex;
+	flex-direction: column;
+	gap: 63px;
```

Comment:
try to avoid odd values in px here and in other places even if they are described in the figma layout